docker-cloud/cloud-swarm/index.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Work with swarms in Docker Cloud
+keywords: swarm mode, swarms, orchestration Cloud, fleet management
+title: Swarms in Docker Cloud
+---
+
+<b>Note</b>: All Swarm management features in Docker Cloud are free while in Beta.
+{: style="text-align:center" }
+
+--------
+Docker Cloud now allows you to connect to clusters of Docker Engines running in v1.13 [Swarm Mode](/engine/swarm/).
+
+## Enable Swarm Mode in Docker Cloud
+
+By default, Docker Cloud allows you to manage [node clusters](../infrastructure/), but you can opt in to use the Beta Swarm Mode
+features. Swarm management replaces the node cluster management features when
+the Swarm Mode interface is enabled.
+
+Click the **Swarm Mode** toggle to enable the swarm mode interface.
+
+You can switch between node cluster and swarm mode at any time, and enabling swarm mode does _not_ remove or disconnect existing node clusters.
+
+Swarm mode is free while in preview.
+
+## Swarm mode and organizations
+
+If you are using Docker Cloud in an [organization](../orgs/), you can use Swarm
+Mode to access any Docker swarms available to your organization. Members of the
+`owners` team grant each team in an organization access to the swarms they need.
+If necessary, you can create new teams to manage beta swarm access.
+
+If you are using swarm mode as a member of a team other than the `owners` team,
+you will only see the swarms that you have been granted access to.
+
+If you are a member of the `owners` team, you must switch to the Swarm Mode
+Docker Cloud interface to grant teams access to your organization's swarms.
+Swarms only appear in the [resource management](../orgs/#/set-team-permissions)
+screens for teams when in the swarm mode interface.
+
+## Register an existing swarm
+
+At this time, you cannot _create_ swarms from within Docker Cloud. However you can register existing swarms as part of the beta functionality.
```

Comment:
This is (hopefully) getting released next week, so this line won't be around long.